sklearn/metrics/cluster/_supervised.py

```diff
@@ -383,6 +390,11 @@ def adjusted_rand_score(labels_true, labels_pred):
 
     .. [wk] https://en.wikipedia.org/wiki/Rand_index#Adjusted_Rand_index
 
+    .. [Chacon22] J. E. Chacón and A. I. Rastrojo, Minimum adjusted Rand index
```

Comment:
The 'o' in the brackets should be an acute accented 'o', (i.e. 'ó') but that causes sphinx errors (see failed builds in earlier commits). Is there a good work-around, or should we just drop the accent for now?

Reply:
Let's drop the accent for the moment.

Reply:
Could we add instead the `:doi:` directive?

Reply:
Thank you for finding this reference (https://github.com/scikit-learn/scikit-learn/issues/8166#issuecomment-1196000573).

I also think we should make sure new references are added using either the `:doi:` or the `:arxiv:` directive.